crankshaft-monitor/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+//! Crate for monitoring crankshaft events.
+use std::net::SocketAddr;
+
+use proto::{Event, monitor_service_server::MonitorServiceServer};
+use server::CrankshaftMonitorServer;
+use tokio::sync::broadcast;
+
+/// protobuf module.
```

Comment:
I think we can be a little more descriptive with this comment (and we should move it to the module itself), something like:

```
//! The module for the generated protobuf code.
```